package.json

```diff
@@ -67,6 +67,7 @@
     "eslint-plugin-react-hooks": "^4.6.0",
     "jsdoc": "^4.0.0",
     "jsdoc-tsimport-plugin": "^1.0.5",
+    "jsdom": "^22.0.0",
```

Comment:
By default, `vitest` runs tests in a node environment. However, PASS runs in a browser environment. JSDom gives a light-weight browser implementation that can be used in testing. It's not perfect, but it's better than nothing.